dashboard.py

```
@@ -9923,11 +10116,11 @@ def _build_energy_rows(source_df, measured_only=False):
                     ),
                     "Concurrencies": st.column_config.NumberColumn(
                         "Concurrencies",
-                        help="Number of different concurrent user load levels tested (e.g., 1, 50, 100, 200, 300, 400, 500, 650 users). Each level runs for ~10 minutes.",
+                        help="Number of different concurrent user load levels tested (e.g., 1, 50, 100, 200, 300, 400, 500, 650 users). Each level runs for ~7.5 min (3.4+) or ~10 min (older).",
                     ),
                     "Benchmark Duration (min)": st.column_config.NumberColumn(
                         "Benchmark Duration (min)",
-                        help="Total benchmark duration in minutes. Calculated as: Number of Concurrencies × 10 minutes per level.",
+                        help="Total benchmark duration in minutes. Calculated as: Number of Concurrencies × duration per level (7.5 or 10 min).",
```

Ruff (0.15.18)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)